test/acceptance/specs/EmailConfirmedSpec.scala

```diff
@@ -116,8 +115,8 @@ class EmailConfirmedSpec extends AcceptanceTestSpec
 
       Then("the user should be on 'Email confirmed' page upon successfully updating the email")
       verifyCurrentPage(EmailConfirmedPage)
-      assertIsTextVisible(EmailConfirmedPage.verifyEmailConfirmedTextPartOne)("Your new email address will be active in 2 hours.")
-      assertIsTextVisible(EmailConfirmedPage.verifyEmailConfirmedTextPartTwo)("Until then we will send CDS emails to the email address you were using previously.")
+      assertIsTextVisible(EmailConfirmedPage.verifyEmailConfirmedTextPartOne)("Your email address b@a.com will be active in 2 hours.")
```

Comment:
Where is test for the second scenario when previous email is not there. if its there then ignore